client/src/main/js/controller/CalendarioController.js

```diff
@@ -136,5 +136,28 @@
             return [];
         }
 
+        /**
+         * Modifica os botões do calendário para terem os estilos seguindo padrão
+         * material, adicionando a classe md-button.
+         * Tem o timeout para a função ser executada assim que a página terminar de ser renderizada.
+         * 
+         * Caso a função seja invocada antes da renderização do calendário, a lista de botões
+         * vai ser vazia, e a função entra em delay novamente para ser re-executada.
+         */
+        function transformaBotoesCalendario() {
+            setTimeout(() => {
```

Comment:
Sem tempo no setTimeout? Então pode ser retirado. Inclusive, pesquisa sobre callback de início do calendário. @velmer mencionou isso uma vez